src/status_im/contexts/wallet/events.cljs

```diff
@@ -163,6 +168,15 @@
   [{:keys [db]}]
   {:db (update-in db [:wallet :ui] dissoc :scanned-address)})
 
+(rf/reg-event-fx :wallet/scan-qr-code-success
```

Comment:
for me it's fine in either, but should these scanner related events live closer to the generic scanner where they are used/defined for? 🤔 
cc @briansztamfater @ulisesmac wdyt?

Reply:
Agree! It shouldn't be linked to wallet but to the scanner itself instead

Reply:
Agree, maybe we can have a wallet event that opens the bottom sheet that would be dispatched from the scanner event?

Reply:
Done